reactioncommerce/reaction
# Fix Setting product visibility is not reflected in the Product Grid reactively

This resolves #2522

### Steps to Test
1. Create a product, set it to visible and publish
1. Observe it in the Product grid in an incognito browser, as non-admin
1. Set it to "Private" and publish
1. Observe that the disappears from the grid
1. Set it to "Public" and publish 
1. Observe that it comes back on the grid reactively.
